packages/lesswrong/server/editor/utils.tests.ts

```diff
@@ -0,0 +1,57 @@
+import { createDummyPost, createDummyUser } from "../../testing/utils";
+import { testStartup } from "../../testing/testMain";
+import Revisions from "../../lib/collections/revisions/collection";
+import { Posts } from "../../lib/collections/posts";
+import { runQuery } from "../vulcan-lib";
+import { syncDocumentWithLatestRevision } from "./utils";
+
+testStartup();
+
+async function updatePost(user: DbUser, postId: string, newMarkup: string) {
+  const query = `
+    mutation PostsEdit {
+      updatePost(
+        selector: {_id:"${postId}"},
+        data: {contents: {originalContents: {type: "ckEditorMarkup", data: "${newMarkup}"}}}
```

Comment:
I assume you don't have to worry about injection vulnerabilities in test handlers, but flagging that if this was anywhere else it might be a bit dubious.

(On reflection, maybe it's just fine, given that it's targeting a publicly-accessible API, so you're not actually escalating privileges.)